mobile/src/auth/refresh-token-storage.ts

```diff
@@ -34,8 +35,33 @@ export class RefreshTokenFilesystemStorage implements RefreshTokenStorage {
                 keychainService: 'ArcadiaAssistant',
                 kSecAccessControl: 'kSecAccessControlTouchIDCurrentSet',
```

Comment:
What happens if it's iOS with IsSensorAvailable = false. So you'd have a combination of touchId=false and this kSecAccessControl.

Reply:
Good question. Will check tomorrow.